Formula/pypy.rb

```diff
@@ -67,12 +79,15 @@ def install
       system python, buildpath/"rpython/bin/rpython",
              "-Ojit", "--shared", "--cc", ENV.cc, "--verbose",
              "--make-jobs", ENV.make_jobs, "targetpypystandalone.py"
+
+      with_env(PYTHONPATH: buildpath) do
+        system "./pypy-c", buildpath/"lib_pypy/pypy_tools/build_cffi_imports.py"
+      end
```

Comment:
May be a no-op as I command may be run at end of `targetpypystandalone.py`. Log output looks like everything is already built:
```
./pypy-c
/private/tmp/pypy-20210813-95966-1txhgk9/pypy2.7-v7.3.5-src/lib_pypy/pypy_tools/build_cffi_imports.py

*  _ctypes._ctypes_cffi already built
*  _ssl already built
*  sqlite3 already built
*  audioop already built
*  _tkinter already built
*  curses already built
*  syslog already built
*  gdbm already built
*  grp already built
*  resource already built
calling create_cffi_import_libraries with "embed_dependencies" False
```

Reply:
I'll just leave it for now as it is part of official build steps and anything that is skipped doesn't really impact build time.